docs/tutorials/custom_experiment.rst

```diff
@@ -0,0 +1,150 @@
+
+Optionally the following methods can also be overridden in the subclass to
+allow configuring various experiment and execution options
+
+- :meth:`.BaseExperiment._default_experiment_options`
+  to set default values for configurable option parameters for the experiment.
+
+- :meth:`.BaseExperiment._default_transpile_options`
+  to set custom default values for the ``qiskit.transpile`` used to
+  transpile the generated circuits before execution.
+
+- :meth:`.BaseExperiment._default_run_options`
+  to set default backend options for running the transpiled circuits on a backend.
+
+- :meth:`.BaseExperiment._default_analysis_options`
+  to set default values for configurable options for the experiments analysis class.
+  Note that these should generally be set by overriding the :class:`.BaseAnalysis`
+  method :meth:`.BaseAnalysis._default_options` instead of this method except in the
+  case where the experiment requires different defaults to the used analysis class.
+
+- :meth:`.BaseExperiment._transpiled_circuits`
+  to override the default transpilation of circuits before execution.
+
+- :meth:`.BaseExperiment._metadata`
+  to add any experiment metadata to the result data.
+
+Furthermore, some characterization and calibration experiments can be run with restless
+measurements, i.e. measurements where the qubits are not reset and circuits are executed
+immediately after the previous measurement. Here, the :class:`.RestlessMixin` can help
+to set the appropriate run options and data processing chain.
+
+Analysis Subclasses
+~~~~~~~~~~~~~~~~~~~
+
+To create an analysis subclass, one only needs to implement the abstract
+:meth:`.BaseAnalysis._run_analysis` method. This method takes a
+:class:`.ExperimentData` container and kwarg analysis options. If any
+kwargs are used the :meth:`.BaseAnalysis._default_options` method should be
+overriden to define default values for these options.
+
+The :meth:`.BaseAnalysis._run_analysis` method should return a pair
+``(results, figures)`` where ``results`` is a list of
+:class:`.AnalysisResultData` and ``figures`` is a list of
+:class:`matplotlib.figure.Figure`.
+
+The :mod:`qiskit_experiments.data_processing` module contains classes for
+building data processor workflows to help with advanced analysis of
+experiment data.
+
+==================================
+Subclassing an Existing Experiment
+==================================
+
+This document will take you step-by-step through the process of subclassing an existing experiment in the Qiskit Experiment module.
+The example in this guide focuses on adjusting the FineAmplitude experiment to calibrate on higher order transitions.
+However, a similar process can be followed for other experiments.
+
+The FineAmplitude Experiment
+============================
+
+The ``FineAmplitude`` calibration experiment repeats N times per gate with a pulse to amplify the under-/over-rotations in the gate to determine the optimal amplitude.
```

Comment:
To my understanding `FineAmplitude` is a characterization experiment, while the calibration version is `FineAmplitudeCal`.

Reply:
@coruscating Please note this comment

Reply:
You're right, and I've updated the characterization experiment docstring so that it doesn't say calibration experiment anymore.